src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.targets

```diff
@@ -626,6 +626,11 @@ Copyright (c) .NET Foundation. All rights reserved.
     <RuntimeHostConfigurationOption Include="System.Threading.ThreadPool.MaxThreads"
                                     Condition="'$(ThreadPoolMaxThreads)' != ''"
                                     Value="$(ThreadPoolMaxThreads)" />
+
+    <RuntimeHostConfigurationOption Include="System.Xml.XmlResolver.IsNetworkingEnabledByDefault"
```

Comment:
The STJ property controlling the `System.Text.Json.JsonSerializer.IsReflectionEnabledByDefault` property was named `JsonSerializerIsReflectionEnabledByDefault`. Echoing that naming convention, shouldn't this also be named `XmlResolverIsNetworkingEnabledByDefault`?

Typically this needs to go through API review but maybe we can fast-track this. cc @eerhardt @bartonjs @terrajobst

Reply:
We've not done API review on these or they would be consistent.

The `XXXSupport` is a prevalent theme for things that disable support for something

`DebuggerSupport`, `EventSourceSupport`, `HttpActivityPropagationSupport`, `MetadataUpdaterSupport`, `NullabilityInfoContextSupport`, `CustomResourceTypesSupport`, `DynamicCodeSupport`, `BuiltInComInteropSupport`,...

Reply:
One difference between the existing XXXSupoort flags and the JsonSerializerIsReflectionEnabledByDefault flag is that the XXXSupport flags completely remove the ability for doing XXX. Where the JsonSerializer is still capable of doing reflection, it just won’t do it by default when the flag is set. Thus the verbose naming (it’s trying to be correct/precise).

I don’t know exactly which case the new XML flag falls into, but if the XmlResolver can still use networking even if the flag is set, I wouldn’t name it XXXSupport. Because it can still do it, even though the flag is set.

Reply:
@Jozkee @krwq do you know which side `IsNetworkingEnabledByDefault` is on so we can finalize the name?

Reply:
I'm not familiar with the code, but looking at [this change](https://github.com/dotnet/runtime/pull/84169/files#diff-965a740bf9db42c616b767e453dc3051d09e03dc2fe8cf797c5b7e38ef7f9cbfR626-R630) suggests that it falls on the same boat as the STJ switch. It picks the `XmlUrlResolver` implementation if the switch is turned on, which is a public class. The behavior of `XmlUrlResolver` itself doesn't seem to be affected by the feature switch.

Reply:
@eiriktsarpalis @eerhardt if this updated name looks good, could you please merge? I'm off work through end of August. It would be nice to have this in 8.0.

Reply:
Thanks!